framework/include/materials/ADMaterial.h

```diff
@@ -17,16 +17,22 @@
   using ADMaterial<compute_stage>::_qp;                                                            \
   using ADMaterial<compute_stage>::_ad_grad_zero;                                                  \
   using ADMaterial<compute_stage>::_ad_zero;                                                       \
-  using ADMaterial<compute_stage>::coupledComponents
+  using ADMaterial<compute_stage>::_qrule;                                                         \
+  using ADMaterial<compute_stage>::_JxW;                                                           \
+  using ADMaterial<compute_stage>::_coord;                                                         \
+  using ADMaterial<compute_stage>::_assembly;                                                      \
+  using ADMaterial<compute_stage>::_mesh;                                                          \
```

Comment:
These will conflict with #12649...it's a race!

Reply:
As you can see from the constantly failing tests I'm merely limping along...